napari/utils/info.py

```diff
@@ -29,7 +29,14 @@ def _linux_sys_name():
                 return f'{data["NAME"]} {data["VERSION_ID"]}'
             return f'{data["NAME"]} (no version)'
 
-    try:
+    return _linux_sys_name_lsb_release()
+
+
+def _linux_sys_name_lsb_release():
+    """
+    Try to discover linux system name base on lsb_release command output
+    """
+    with contextlib.suppress(subprocess.CalledProcessError):
         res = subprocess.run(
```

Comment:
something went wrong here I think, or the diff is messed up.

Reply:
I have changed `try: ... except: pass` on.`suppress`